bluetooth/src/commonMain/kotlin/device/AdvertisementData.kt

```diff
@@ -40,4 +40,15 @@ val BaseAdvertisementData.description: String get() = listOfNotNull(
     if (serviceData.isEmpty()) null else { "ServiceData: ${serviceData.entries.joinToString(",") { (uuid, data) -> "[${uuid.uuidString} : $data]" } }" },
     "TxPowerLevel: $txPowerLevel",
     "IsConnectable: $isConnectable"
-).joinToString("\n").ifEmpty { "Empty Advertisement Data" }
+).joinToString("\n")
+
+internal data class ConnectableEmptyAdvertisementData(
+    override val name: String? = null
```

Comment:
Not sure if it should be in the constructor

Reply:
I provide here name, as it's available (but the rest is not)